lib/nel.js

```diff
@@ -49,9 +49,27 @@ var fs = require("fs");
 var path = require("path");
 var spawn = require("child_process").spawn;
 
+function findFile(file, dir) {
+    var result = fs.readdirSync(dir).find(result => result.indexOf(file) >= 0);
+    if (result) return path.resolve(result);
+    return findFile(file, path.join(dir, ".."));
+}
+
+function getBabelConfig(path) {
```

Comment:
Just a style comment. To avoid bugs hard to find, I prefer not to shadow `path`.